build.gradle

```diff
@@ -50,17 +50,25 @@ dependencies {
     // The classes of "embulk-spi" are excluded from "embulk-core" below for the consistency as a JAR file.
     // But, they should be included in the compile classpath.
     compileOnly "org.embulk:embulk-spi:0.11"
+    compileOnly "org.slf4j:slf4j-api:2.0.13"
+    compileOnly "org.msgpack:msgpack-core:0.8.24"
```

Comment:
Just for interest. Why need the `msgpack-core` in this PR?

Reply:
It has been included in `embulk-spi` implicitly. There is no substantial changes. Not needed literally.

Wanted to include this just to be explicit.